### DISCUSSION TITLE
What did Old Mother Hubbard find in her cupboard?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#761] [Galaxy Trivia] [answered]
**Bonus:** How many places did Old Mother visit that day?

**Comment by peterbaccaro** [comment, marked as answer]
Old Mother Hubbard found **nothing** in her cupboard as it was bare.

Based on the original nursery rhyme she visited **11 places**.

1. Her cupboard
2. The baker's
3. The undertaker's
4. Her own home (to get a clean dish)
5. The ale-house
6. The tavern
7. The fruiterer's
8. The tailor's
9. The cobbler's
10. The seamstress's
11. The hosier's

The original nursery rhyme is as follows:

Old Mother Hubbard,
Went to the cupboard,
To give the poor dog a bone:
When she came there,
The cupboard was bare,
And so the poor dog had none.

She went to the baker’s
To buy him some bread,
But when she came back,
The poor dog was dead.

She went to the undertaker’s
To buy him a coffin,
But when she came back,
The poor dog was laughing.

She took a clean dish,
To get him some tripe,
But when she came back,
He was smoking his pipe.

She went to the ale-house,
To get him some beer,
But when she came back,
The dog sat in a chair.

She went to the tavern,
For white wine and red,
But when she came back,
The dog stood on his head.

She went to the fruiterer’s,
To buy him some fruit,
But when she came back,
He was playing the flute.

She went to the tailor’s,
To buy him a coat,
But when she came back,
He was riding a goat.

She went to the cobbler's,
To buy him some shoes,
But when she came back,
He was reading the news.

She went to the seamstress,
To buy him some linen,
But when she came back,
The dog was spinning.

She went to the hosier’s,
To buy him some hose,
But when she came back,
He was dressed in his clothes.

The dame made a curtsy,
The dog made a bow;
The dame said, "Your servant,"
The dog said, "Bow-wow."
